simplified dockerfile

belkone opened the pull request:
continuation of the thread https://github.com/mkaczanowski/packer-builder-arm/pull/22 with @dbast ;) 
Inspired by https://github.com/mkaczanowski/packer-builder-arm/issues/17

mkaczanowski commented:
@dbast can you check this PR? (seems you guys have quite a conversation on the docker stuff :) )

dbast commented:
Tested on a fresh Mac... works fine! :) 

The register stuff indeed removes the need to run `docker run --rm --privileged multiarch/qemu-user-static --reset -p yes` before running the image. So now only one single command to use the Docker image :)

We can now also remove installing `binfmt-support` in the Dockerfile as this now seems also be provided via the register trick. Also tested that here.

belkone commented:
> The register stuff indeed removes the need to run `docker run --rm --privileged multiarch/qemu-user-static --reset -p yes` before running the image. So now only one single command to use the Docker image :)

Yeah, I updated README about it in first commit ;)

> We can now also remove installing `binfmt-support` in the Dockerfile as this now seems also be provided via the register trick. Also tested that here.

That's true, I removed this package, thanks :)
